e2e/support/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@cypress//:index.bzl", "cypress_web_test")
+load("@npm//@bazel/typescript:index.bzl", "ts_library")
+
+# You must create a cypress plugin in order to boot a server to serve your application. It can be written as a javascript file or in typescript using ts_library or ts_project.
+ts_library(
```

Comment:
@subotic when running `make test`, it still complains about the missing supportFile:

 > The support file is missing or invalid.

> Your `supportFile` is set to `/private/var/tmp/_bazel_tobiasschweizer/33763300ac71cf3e265fe252a97576ea/sandbox/darwin-sandbox/40/execroot/io_dasch_dsp_api/bazel-out/darwin-fastbuild/bin/e2e/test.sh.runfiles/io_dasch_dsp_api/e2e/support/commands.ts`, but either the file is missing or it's invalid. The `supportFile` must be a `.js`, `.ts`, `.coffee` file or be supported by your preprocessor plugin (if configured).

> Correct your `./cypress.json`, create the appropriate file, or set `supportFile` to `false` if a support file is not necessary for your project.

> Or you might have renamed the extension of your `supportFile` to `.ts`. If that's the case, restart the test runner.

I think there is still some export missing

Reply:
and I think there will be the same issue integrating the spec files from "e2e/integration".

However, it works when I do "npm t" (`cypress run --config-file cypress.json --browser chrome`)